test/runtests.jl

```diff
@@ -1,16 +1,19 @@
+push!(Base.LOAD_PATH,joinpath(dirname(@__FILE__),"..",".."))
```

Comment:
That is this? Nit: spaces after `,`

Reply:
this is me forgetting to remove debug lines, I was waiting a bit more to remove since it was already there (and very helpful)

Reply:
with this you just include runtests